src/huggingface_hub/hf_api.py

```diff
@@ -193,8 +209,8 @@ class ModelInfo:
             List of tags.
         pipeline_tag (`str`, *optional*):
             Pipeline tag to identify the correct widget.
-        siblings (`List[Dict]`, *optional*):
-            list of files that constitute the Space
+        siblings (`List[RepoFile]`, *optional*):
+            list of ([`huggingface_hub.hf_api.RepoFile`]) objects that constitute the model
```

Comment:
```suggestion
            list of ([`huggingface_hub.hf_api.RepoFile`]) objects that constitute the model.
```

Reply:
arf yeah, i guess this docstring documents the class, not its init. So the change is correct